pkgs/tools/networking/zerotierone/default.nix

```diff
@@ -65,6 +65,19 @@ in stdenv.mkDerivation {
 
   enableParallelBuilding = true;
 
+  # Ensure Rust compiles for the right target
+  env.CARGO_BUILD_TARGET = stdenv.hostPlatform.rust.rustcTarget;
+
+  # Cargo won't compile to target/release but to target/<RUST_TARGET>/release when a target is
+  # explicitly defined. The build-system however expects target/release. Hence we just symlink from
+  # the latter to the former.
+  preBuild = ''
+    mkdir --parent -- rustybits/target/release
+    ln --relative --symbolic -- \
```

Comment:
```suggestion
    ln -rs \
```

Reply:
Intentionally left long, see https://github.com/NixOS/nixpkgs/pull/314824#discussion_r1615187793

Reply:
I think these numbers are mixed up.

Reply:
As far as I'm concerned, we have mutual understanding. If there is a strong pressure to go for the shortened flags, I will comply. If this is mere a suggestion from your side, I acknowledge and respectfully defy it 😄